nb/updatecenters/src/org/netbeans/modules/updatecenters/resources/Bundle.properties

```diff
@@ -25,13 +25,16 @@ OpenIDE-Module-Short-Description=Declares NetBeans autoupdate centers.
 
 Services/AutoupdateType/distribution-update-provider.instance=NetBeans Distribution
 Services/AutoupdateType/pluginportal-update-provider.instance=NetBeans Plugin Portal
+Services/AutoupdateType/legacy-pluginportal-update-provider.instance=Legacy NetBeans Plugin Portal
 Services/AutoupdateType/82pluginportal-update-provider.instance=NetBeans 8.2 Plugin Portal
 
 #NOI18N
 URL_Distribution=@@metabuild.DistributionURL@@
 #NOI18N
 URL_PluginPortal=@@metabuild.PluginPortalURL@@
 #NOI18N
+URL_LegacyPluginPortal=@@metabuild.LegacyPluginPortalURL@@
```

Comment:
Does this need to be set somewhere?

Just looking at the other metadata its also used:
https://github.com/apache/netbeans/blob/312f983bf6627e91bdfb7526e67302975d3f7f80/nb/updatecenters/build.xml#L31
and
https://github.com/apache/netbeans/blob/6c3e8d1eee5c8ca49c11027065ca90c8abcbd920/nbbuild/antsrc/org/netbeans/nbbuild/ReleaseJsonProperties.java#L177